ci/test_flake8.sh

```diff
@@ -56,6 +59,7 @@ espnet/nets/pytorch_backend/e2e_asr.py
 espnet/nets/pytorch_backend/e2e_asr_mix.py
 espnet/nets/pytorch_backend/e2e_asr_transducer.py
 espnet/nets/pytorch_backend/e2e_asr_transformer.py
+espnet/nets/pytorch_backend/e2e_st.py
```

Comment:
Can you remove those files from the blacklist (at least this file)? I think nobody can do this later.

Reply:
I think we cannot pass tests without this (even for ASR).

Reply:
We try to remove from these list.
Since we do not want to add new files, if you update your docstrings, it is very helpful.

Reply:
You can pass if you add docstrings.

Reply:
OK. But this is not a problem for RNN-T...

Reply:
RNNT and flake8 docstrings were merged at the same time and I added that against my will... #1107